.circleci/config.yml

```diff
@@ -0,0 +1,61 @@
+defaults: &defaults
+  working_directory: /tmp/skt
+  steps:
+    - checkout
+    - run:
+        name: Install skt using pip
+        command: pip install .
+    - run:
+        name: Run Python linters and tests
+        command: |
+          flake8 --show-source .
+          pylint tests
+          python -m unittest discover tests
+    - run:
+        name: Clone and prepare repo
+        command: |
+          mkdir -p /tmp/skt_test/workdir
+          git clone --depth 1 git://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git /tmp/skt_test/workdir
```

Comment:
I agree with @dzickusrh that we need to fix the kernel version with which we are testing. We're testing skt here, not the kernel, after all. And then we don't want patching to break once it stops applying.

Another thing is I think we should be cloning the kernel using skt itself (with the `merge` command). What do you think?